content/en/core/overview/architecture.md

```diff
@@ -32,13 +32,70 @@ A free and open source SQL database that we use for analytics queries for displa
+
+It uses the [Ngx-translate](https://github.com/ngx-translate/core) for Angular framework to automatically translate the application labels. See [here](https://docs.communityhealthtoolkit.org/apps/reference/translations/) how to manage translations.
+
+It uses [Karma](https://github.com/karma-runner/karma) and [MochaJS](https://mochajs.org/) to run the [application’s unit tests](https://github.com/medic/cht-core/tree/master/webapp/tests)
+
+It uses [Protractor](https://www.protractortest.org/#/) to run the [application’s e2e tests](https://github.com/medic/cht-core/tree/master/tests/e2e)
+
+It uses [Less](http://lesscss.org/) as a CSS preprocessor.
+
+##### Structure
+
+The CHT Web Application has the following high level structure: 
+
+- **/js**: Contains the vanilla JavaScript scripts, for example: Enketo widgets, MomentJS locales, etc.
+- **/ts**: Contains the Angular application source code which uses TypeScript.
+  - **/actions**, **/effects**, **/reducers** and **/selectors**: Contain the implementation for the application’s reactive state which uses [NgRx](https://ngrx.io) framework.
+  - **/components**, **/directives**, **/pipes**, **/providers** and **/services**: Contain the reusable elements from [Angular](https://angular.io/) framework.
+  - **/modals**: Contains the all application’s modals components.
+  - **/modules**: Contains the application’s modules, each of them has components that are associated to the modules’ routing.
+- **/css**:  Contains the application’s style files. It uses [Less](http://lesscss.org/) as a CSS preprocessor. 
+- **/fonts**: Contains the application fonts.
+- **/img**: Contains the application static images.
+#### App Management
+[App Management](https://docs.communityhealthtoolkit.org/apps/features/admin/) is an interface for non-technical administrative users to manage users and settings.
+
+Click [here](https://github.com/medic/cht-core/tree/master/admin) to see the application source code.
+
+##### Technology
+App Management is a single page application built with [AngularJS](https://angularjs.org) framework and implements [Redux](https://github.com/reduxjs/redux) to manage a reactive state.
+
+It uses [PouchDB](https://pouchdb.com) to store the data and it replicates changes back and forth in the background with the server CouchDB.
+
+It uses [Angular Translate](https://github.com/angular-translate/angular-translate) for AngularJS to automatically translate the application labels. See [here](https://docs.communityhealthtoolkit.org/apps/reference/translations/) how to manage translations.  
+
+It uses [Karma](https://github.com/karma-runner/karma) and [MochaJS](https://mochajs.org/) to run the [application’s unit tests](https://github.com/medic/cht-core/tree/master/admin/tests)
+
+It uses [Protractor](https://www.protractortest.org/#/) to run the [application’s e2e tests](https://github.com/medic/cht-core/tree/master/tests/e2e)
+
+It uses [Less](http://lesscss.org/) as a CSS preprocessor.
```

Comment:
It's probably worth using a table here as well so it's consistent with the above.

Reply:
Done